types/mercadopago-sdk-js/modules/bricks.d.ts

```diff
@@ -1,328 +1,335 @@
-    zip_code: string;
-    federal_unit: string;
-    city: string;
-    street_name: string;
-    street_number: string;
-    neighborhood: string;
-  }
-
-  interface PayerAPI {
-    email?: string;
-    identification?: PayerIdentification;
-    address?: PayerAddressAPI;
-    first_name?: string;
-    last_name?: string;
-  }
-
-  enum PaymentType {
-    CREDIT_CARD = "credit_card",
-    DEBIT_CARD = "debit_card",
-    ICKET = "ticket",
-    BANK_TRANSFER = "bank_transfer",
-    WALLET_PURCHASE = "wallet_purchase",
-    ONBOARDING_CREDITS = "onboarding_credits",
-    ATM = "atm",
-    ATM_ONLINE = "atm_online",
-    NONE = "",
-  }
-
-  interface SavedCardPayer {
-    type: "customer";
-    id: string;
-  }
-
-  interface CardFormData {
-    token: string;
-    issuer_id: string | null;
-    payment_method_id: string;
-    transaction_amount: number;
-    payment_method_option_id?: string;
-    processing_mode?: string;
-    installments: number;
-    payer: PayerAPI;
-  }
-
-  interface SavedCardFormData {
-    token: string;
-    issuer_id: string | null;
-    payment_method_id: string;
-    transaction_amount: number;
-    payment_method_option_id?: string;
-    processing_mode?: string;
-    installments: number;
-    payer: SavedCardPayer;
-  }
-
-  interface TicketFormData {
-    transaction_amount: number;
-    payment_method_id: string;
-    payer: PayerAPI;
-  }
-
-  interface AdditionalSavedCardFormData {
-    bin: string;
-  }
-
-  interface AdditionalCardFormData {
-    bin: string;
-  }
-
-  type AdditionalTicketFormData = Record<string, unknown>;
-
-  type AdditionalPaymentFormData =
-    | AdditionalCardFormData
-    | AdditionalSavedCardFormData
-    | AdditionalTicketFormData;
-
-  interface PaymentFormData {
-    paymentType: PaymentType;
-    selectedPaymentMethod: PaymentType;
-    formData?: CardFormData | SavedCardFormData | TicketFormData | null;
-    additionalData?:
-    | AdditionalSavedCardFormData
-    | AdditionalCardFormData
-    | AdditionalTicketFormData
-    | null;
-  }
-
-  interface BrickController {
-    unmount: () => void;
-    getFormData: () => Promise<CardFormData | PaymentFormData>;
-    getAdditionalData: () => Promise<
-      AdditionalCardFormData | AdditionalPaymentFormData
-    >;
-  }
-
-  type BrickTypes = 'cardPayment' | 'payment' | 'statusScreen' | 'wallet';
-
-  interface Bricks {
-    isInitialized(): boolean;
-    create(
-      brick: BrickTypes,
-      containerId: string,
-      settings: BrickSettings
-    ): Promise<BrickController>;
-  }
+    interface BrickError {
+        type: 'non_critical' | 'critical';
+        cause: string;
+        message: string;
+    }
+
+    interface BrickCallbacks {
+        onReady?: () => void;
+        onError?: (error: BrickError) => void;
+    }
+
+    interface Submit<BrickType> {
+        onSubmit: (
+            formData: CardFormData | PaymentFormData,
+            param2?: AdditionalCardFormData | AdditionalPaymentFormData,
```

Comment:
Na documentação nós chamamos esse parâmetro de `additionalData`
```suggestion
            additionalData?: AdditionalCardFormData | AdditionalPaymentFormData,
```